molgenis-data-mapper/src/main/resources/templates/view-attribute-mapping-feedback.ftl

```diff
@@ -36,6 +37,8 @@
 		<tbody>
 			<#list feedbackRows as feedbackRow>
 				<tr>
+				    <#-- Dataexplorer can't be initialized with query at the moment, for forward compatibility already construct URL -->
+				    <td><a class="btn btn-default btn-xs" href="javascript:window.location='${dataexplorerUri?html}?entity=${source?html}&q=' + molgenis.createRsqlQuery([{field: '${feedbackRow.sourceEntity.getEntityMetaData().getName()?html}', operator: 'EQUALS', value: '${feedbackRow.sourceEntity.getIdValue()?string?html}' }]);" role="button"><span class="glyphicon glyphicon-search"></span></a></td>
```

Comment:
I get `http://localhost:8080/menu/main/dataexplorer?entity=LifeLines&q=LifeLines==1`, shouldn't that be `http://localhost:8080/menu/main/dataexplorer?entity=LifeLines&q=ID==1`?